fastlane/download_metadata.swift

```diff
@@ -6,7 +6,26 @@ let glotPressSubtitleKey = "app_store_subtitle"
 let glotPressWhatsNewKey = "v17.6-whats-new"
 let glotPressDescriptionKey = "app_store_desc"
 let glotPressKeywordsKey = "app_store_keywords"
-let baseFolder = "./metadata"
+
+struct Config {
+    let baseFolder: String
+    let baseURLString: String
+
+    static let wordPress = Config(
+        baseFolder: "./metadata",
+        baseURLString: "https://translate.wordpress.org/projects/apps/ios/release-notes/"
+    )
+
+    static let jetpack = Config(
+        baseFolder: "./jetpack_metadata",
+        baseURLString: "https://translate.wordpress.com/projects/jetpack/apps/ios/release-notes/"
+    )
+
+    static func config(for argument: String?) -> Config {
+        guard let argument = argument else { return .wordPress }
+        return argument == "jetpack" ?  .jetpack : .wordPress
+    }
+}
```

Comment:
Had a bit of Swift fun here. But I worry I ended up with something over-engineered...

Reply:
I like it (but it might be because I miss Swift too and playing with it 😅 )